tests/test_primary.py

```diff
@@ -578,30 +579,46 @@ def test_20_validate_time_attestation(self):
           original_time_attestation, DATATYPE_TIME_ATTESTATION,
           private_key=TestPrimary.key_timeserver_pri, resign=True)
 
+    # Check expected base conditions before updating time:
+    # The only timeserver times registered should be two "now"s added during
```

Comment:
Could it be that this is a copy-paste mistake (from `test_secondary.py`)? You say that it should have "two" registered times but are asserting for `len==1` below. Maybe I'm missing something.

Reply:
Fixed both now, thanks. :)